### PR TITLE
Merge Stage to Test

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html [hunk 1]
```diff
@@ -431,421 +431,434 @@ <h3>{{activeCompany?.baseCurrencySymbol}} {{overallTotal | giddhCurrency}}</h3>
...
+                                    <td>{{i+1}}.</td>
+                                    <td class="min-wid-product">
+                                        <div *ngIf="activeRow === i">
+                                            <sh-select name="productName{{i}}" #selectDropdown [notFoundLink]="true"
+                                                (noResultsClicked)="onProductNoResultsClicked(i)"
+                                                [isFilterEnabled]="true" [forceClearReactive]="forceClear$ | async"
+                                                [options]="stockList" [(ngModel)]="product.uniqueName"
+                                                (selected)="selectProduct($event, product);"></sh-select>
+                                            <div class="position-relative wrap-hsn-box">
+                                                <div class="skuCode" (click)="focusSkuNumber();"
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html:470]: Code reverted

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.html [hunk 2]
```diff
@@ -276,6 +276,24 @@ <h4 class="modal-title bg pull-left">{{localeData?.search_here}}</h4>
                                         <ng-container *ngTemplateOutlet="sortTemplate;context: {column: 'nextcrondate'}">
                                         </ng-container>
 
+                                    </div>
+                                </div>
+                            </th>
+                            <th>
+                                <div class="d-flex">
+                                    <div class=""> {{localeData?.last_invoice_date}}</div>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/invoice/recurring/recurring.component.html:284]: remove blank class

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.html [hunk 3]
```diff
@@ -276,6 +276,24 @@ <h4 class="modal-title bg pull-left">{{localeData?.search_here}}</h4>
...
+                                    <div class="icon-pointer">
+                                        <ng-container *ngTemplateOutlet="sortTemplate;context: {column: 'lastinvoicedate'}">
+                                        </ng-container>
+                                    </div>
+                                </div>
+                            </th>
+                            <th>
+                                <div class="d-flex">
+                                    <div class=""> {{localeData?.next_invoice_date}}</div>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/invoice/recurring/recurring.component.html:293]: remove blank class

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.html [hunk 4]
```diff
@@ -287,6 +305,15 @@ <h4 class="modal-title bg pull-left">{{localeData?.search_here}}</h4>
                                         <ng-container *ngTemplateOutlet="sortTemplate;context: {column: 'status'}">
                                         </ng-container>
 
+                                    </div>
+                                </div>
+                            </th>
+                            <th>
+                                <div class="d-flex">
+                                    <div class="">{{commonLocaleData?.app_status}}</div>
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/invoice/recurring/recurring.component.html:313]: remove blank class

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-e/gst-template-e.component.html [hunk 5]
```diff
@@ -537,7 +537,7 @@ <h2 *ngIf="fieldsAndVisibility.table.totalDue?.display">7124.00/- </h2>
                     </td> -->
 
                     <!-- <td class="paid-imgWrapper">
-                        <img src="https://apitest.giddh.com/seal/cancelled" class="img-cancel">
+                        <img src="https://api.giddh.com/seal/cancelled" class="img-cancel">
```
Comment (shalinee-walkover) [inline apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-e/gst-template-e.component.html:540]: check, hope its correct

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.scss [hunk 6]
```diff
@@ -47,7 +47,7 @@
     background-color: $white;
     z-index: 1000;
     width: 257px;
-    border: 1px solid $gainsboro;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.scss:50]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/services/common.service.ts [hunk 7]
```diff
@@ -27,15 +27,6 @@ export class CommonService {
             }));
     }
 
-    public GetCurrency(): Observable<BaseResponse<any, any>> {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/services/common.service.ts:30]: @prateek-giddh Please check this code removal

File: apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss [hunk 8]
```diff
@@ -1,9 +1,8 @@
 @import "../../../../../assets/css/color.scss";
-
 .list-item:focus,
 .list-item:focus .list-item {
     color: $white !important;
-    background: $primary;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss:6]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 9]
```diff
@@ -62,7 +62,7 @@
 }
 
 a[vr-item]:focus {
-    background-color: $primary;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.scss:65]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 10]
```diff
@@ -811,6 +827,13 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
      * @memberof HeaderComponent
      */
     public toggleSidebarPane(show: boolean, isMobileSidebar: boolean): void {
+        if(!this.isIpadScreen) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:830]: This code shouldn't be added

File: apps/web-giddh/src/app/shared/on-boarding/on-boarding.component.scss [hunk 11]
```diff
@@ -61,7 +61,7 @@ label .required {
 }
 
 .name-error {
-    color: $scarlet;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/on-boarding/on-boarding.component.scss:64]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/tallysync/completed/completed.component.scss [hunk 12]
```diff
@@ -24,7 +24,7 @@
             box-shadow: none;
             height: 5px;
             margin-bottom: 0px;
-            border: $whisper 1px solid;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/tallysync/completed/completed.component.scss:27]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/tallysync/inprogress/inprogress.component.scss [hunk 13]
```diff
@@ -25,7 +25,7 @@
             box-shadow: none;
             height: 5px;
             margin-bottom: 0px;
-            border: $whisper 1px solid;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/tallysync/inprogress/inprogress.component.scss:28]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss [hunk 14]
```diff
@@ -104,29 +104,29 @@ i.close.icon.clear:hover::after {
 }
 
 .selected-val {
-    border: 1px solid $light-gray !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss:107]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss [hunk 15]
```diff
@@ -104,29 +104,29 @@ i.close.icon.clear:hover::after {
...
 .header.open > input[type="text"] {
-    border-color: $light-gray;
+    border-color: $gainsboro;
 }
 
 .header .bdr > input[type="text"] {
     border: 0;
 }
 .header .bdr > input[type="text"]::-webkit-input-placeholder {
-    color: $nobel !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss:120]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss [hunk 16]
```diff
@@ -142,7 +142,7 @@ i.close.icon.clear:hover::after {
     bottom: 0;
     cursor: pointer;
     background: $solitude;
-    color: $dim-gray;
+    color: $dark-gray;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss:145]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss [hunk 17]
```diff
@@ -171,7 +171,7 @@ button.form-control:focus {
 
 .bottom-border-only {
     border: none !important;
-    border-bottom: 1px solid $light-gray !important;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss:174]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.scss [hunk 18]
```diff
@@ -93,7 +93,7 @@
 }
 .subscription-dropdown li:hover a,
 .subscription-dropdown li:hover {
-    background: $white-smoke;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.scss:96]: @aditi-tiwari17 @shalinee-walkover Please check this code revert

File: apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.scss [hunk 19]
```diff
@@ -1104,6 +1102,11 @@ strong {
 }
 
 /*responsive*/
+@media (max-width: 1199px) {
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/invoice/preview/models/invoice-preview-details/invoice-preview-details.component.scss:1105]: code copied

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html (hunk 1 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.add.component.html:470]: Done

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.html [hunk 20]
```diff
@@ -299,6 +326,15 @@ <h4 class="modal-title bg pull-left">{{localeData?.search_here}}</h4>
                                         <ng-container *ngTemplateOutlet="sortTemplate;context: {column: 'vouchertotal'}">
                                         </ng-container>
 
+                                    </div>
+                                </div>
+                            </th>
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/recurring/recurring.component.html:331]: Done

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.html (hunk 4 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/recurring/recurring.component.html:313]: Done

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.html [hunk 21]
```diff
@@ -287,6 +305,15 @@ <h4 class="modal-title bg pull-left">{{localeData?.search_here}}</h4>
                                         <ng-container *ngTemplateOutlet="sortTemplate;context: {column: 'status'}">
                                         </ng-container>
 
+                                    </div>
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/recurring/recurring.component.html:308]: Done

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.html (hunk 3 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/recurring/recurring.component.html:293]: Done

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.html (hunk 2 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/recurring/recurring.component.html:284]: Done

File: apps/web-giddh/src/app/invoice/recurring/recurring.component.html [hunk 22]
```diff
@@ -276,6 +276,24 @@ <h4 class="modal-title bg pull-left">{{localeData?.search_here}}</h4>
                                         <ng-container *ngTemplateOutlet="sortTemplate;context: {column: 'nextcrondate'}">
                                         </ng-container>
 
+                                    </div>
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/recurring/recurring.component.html:279]: Done

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts [hunk 23]
```diff
@@ -33,6 +33,7 @@ import { InvoiceReceiptActions } from 'apps/web-giddh/src/app/actions/invoice/re
 import { CompanyResponse, ValidateInvoice } from 'apps/web-giddh/src/app/models/api-models/Company';
 import { CompanyActions } from 'apps/web-giddh/src/app/actions/company.actions';
 import { InvoiceAdvanceSearchComponent } from './models/advanceSearch/invoiceAdvanceSearch.component';
+import { BulkExportModal } from './models/bulk-export-modal/bulk-export.component';
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts:36]: Done

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts [hunk 24]
```diff
@@ -33,6 +33,7 @@ import { InvoiceReceiptActions } from 'apps/web-giddh/src/app/actions/invoice/re
 import { CompanyResponse, ValidateInvoice } from 'apps/web-giddh/src/app/models/api-models/Company';
 import { CompanyActions } from 'apps/web-giddh/src/app/actions/company.actions';
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.ts:34]: Done

File: apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.ts [hunk 25]
```diff
@@ -73,14 +73,11 @@ export class DownloadOrSendInvoiceOnMailComponent implements OnInit, OnDestroy {
     ) {
         this.breakpointObserver
         .observe(['(max-width: 768px)'])
-        .pipe(takeUntil(this.destroyed$))
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.ts]: Done

File: apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.ts [hunk 26]
```diff
@@ -73,14 +73,11 @@ export class DownloadOrSendInvoiceOnMailComponent implements OnInit, OnDestroy {
...
         .subscribe((state: BreakpointState) => {
             this.isMobileScreen = state.matches;
         });
 
         this.isErrOccured$ = this.store.pipe(select(p => p.invoice.invoiceDataHasError), distinctUntilChanged(), takeUntil(this.destroyed$));
-        this.voucherDetailsInProcess$ = this.store.pipe(select(p => p.receipt.voucherDetailsInProcess), takeUntil(this.destroyed$));
-        this.voucherPreview$ = this.store.pipe(select(p => p.receipt.base64Data), distinctUntilChanged(), takeUntil(this.destroyed$));
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/preview/models/download-or-send-mail/download-or-send-mail.component.ts:83]: Done

File: apps/web-giddh/src/app/invoice/proforma/proforma-list.component.ts [hunk 27]
```diff
@@ -188,15 +188,12 @@ export class ProformaListComponent implements OnInit, OnDestroy, OnChanges {
...
     /* This will hold common JSON data */
     public commonLocaleData: any = {};
-    /** True if user has voucher list permission */
     public hasVoucherListPermissions: boolean = true;
 
     constructor(private store: Store<AppState>, private proformaActions: ProformaActions, private router: Router, private _cdr: ChangeDetectorRef, private _breakPointObservar: BreakpointObserver, private generalService: GeneralService, private modalService: BsModalService, private commonActions: CommonActions) {
         this.advanceSearchFilter.page = 1;
         this.advanceSearchFilter.count = 20;
         this.advanceSearchFilter.from = moment(this.datePickerOptions.startDate).format(GIDDH_DATE_FORMAT);
-        this.advanceSearchFilter.to = moment(this.datePickerOptions.endDate).format(GIDDH_DATE_FORMAT);
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/proforma/proforma-list.component.ts:198]: Done

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.html [hunk 28]
```diff
@@ -435,6 +435,10 @@
                             <td class="border-none" [attr.colspan]="columnsVisibled - 1">Dummy Product Description 1
                             </td>
                         </tr>
+                        <tr *ngIf="fieldsAndVisibility.table.item?.display && fieldsAndVisibility.table.showDescriptionInRows?.display">
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.html:438]: Done

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.html [hunk 29]
```diff
@@ -482,6 +486,10 @@
                             <td class="border-none" [attr.colspan]="columnsVisibled - 1">Dummy Product Description 2
                             </td>
                         </tr>
+                        <tr *ngIf="fieldsAndVisibility.table.item?.display && fieldsAndVisibility.table.showDescriptionInRows?.display">
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.html:489]: Done

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.html (hunk 28 above)
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.html:438]: Done

File: apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.html (hunk 29 above)
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/invoice/templates/edit-template/out-tempate/templates/gst-template-a/gst-template-a.component.html:489]: Done

File: apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html [hunk 30]
```diff
@@ -1341,31 +1340,6 @@
                                             (blur)="updateExchangeRate(grandTotalInr.value)">
                                     </div>
                                 </div>
-
```
[the remark this answers is not in the document]
Comment (prateek-giddh) [reply apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html:1344]: Done

File: apps/web-giddh/src/app/services/common.service.ts (hunk 7 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/services/common.service.ts:30]: @ravinderwalkover you removed this code from all the branches but somehow it is there on giddh-2.0 branch, therefore, keeping this change.

File: apps/web-giddh/src/app/shared/header/header.component.ts (hunk 10 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/shared/header/header.component.ts]: Done

File: apps/web-giddh/src/app/accounting/journal-voucher/journal-voucher.component.scss [hunk 31]
```diff
@@ -100,7 +100,7 @@
     padding: 0 !important;
     border: 0;
     z-index: 0;
-    border-left: 1px solid $light-gray;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/accounting/journal-voucher/journal-voucher.component.scss:103]: checked, its fine

File: apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.scss [hunk 32]
```diff
@@ -316,7 +316,7 @@ td.border-right-none {
 
 button.btn.btn-filter {
     background: $whisper;
-    border: 1px solid $whisper;
+    border: 1px solid $gainsboro;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.scss:319]: checked, its fine

File: apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.component.scss [hunk 33]
```diff
@@ -288,7 +288,7 @@ tfoot td .table-sec.form-group {
 }
 .trash-icon-delete a {
     color: $nobel;
-    border: 1px solid $nobel;
+    border: 1px solid $gainsboro;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/inventory/components/new-branch-transfer/new.branch.transfer.component.scss:291]: checked, its fine

File: apps/web-giddh/src/app/inventory/jobwork/jobwork.component.scss [hunk 34]
```diff
@@ -62,7 +62,7 @@
     top: 10px;
 }
 .table-border {
-    border: 1px solid $silver;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/inventory/jobwork/jobwork.component.scss:65]: checked, its fine

File: apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.scss [hunk 35]
```diff
@@ -147,15 +147,15 @@
 
 .table-modal tbody td {
     border-top: 0px solid !important;
-    border-right: 1px solid $silver;
+    border-right: 1px solid $gainsboro;
 }
 
 .table-modal thead th {
-    border-right: 1px solid $silver !important;
+    border-right: 1px solid $gainsboro !important;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.scss:154]: checked, its fine

File: apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.scss [hunk 36]
```diff
@@ -147,15 +147,15 @@
...
 }
 
 .table-modal {
-    border-bottom: 1px solid $silver !important;
+    border-bottom: 1px solid $gainsboro !important;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.scss:158]: checked, its fine

File: apps/web-giddh/src/app/ledger/components/ledger-discount/ledger-discount.component.scss [hunk 37]
```diff
@@ -20,7 +20,7 @@
     color: $dim-gray;
     margin-bottom: 10px;
 }
-.my-dropdown-men {
+.my-dropdown-menu {
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/ledger/components/ledger-discount/ledger-discount.component.scss:23]: checked, its fine

File: apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.scss [hunk 38]
```diff
@@ -73,18 +75,14 @@ ng-select .select-drop i.fa.fa-caret-up::before {
...
+    color: $silver;
 }
 
 .notfound-option.active {
-    background: $lavender !important;
+    background: $whisper-color !important;
     padding: 2px 5px !important;
 }
 
-.p-0 {
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.scss:84]: checked, its fine

File: apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.scss (hunk 18 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.scss:96]: checked, its fine

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss (hunk 17 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss:174]: checked, its fine

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss (hunk 16 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss:145]: checked, its fine

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss (hunk 15 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss:120]: checked, its fine

File: apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss (hunk 14 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss:107]: checked, its fine

File: apps/web-giddh/src/app/tallysync/inprogress/inprogress.component.scss (hunk 13 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/tallysync/inprogress/inprogress.component.scss:28]: checked, its fine

File: apps/web-giddh/src/app/tallysync/completed/completed.component.scss (hunk 12 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/tallysync/completed/completed.component.scss:27]: checked, its fine

File: apps/web-giddh/src/app/shared/on-boarding/on-boarding.component.scss (hunk 11 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/shared/on-boarding/on-boarding.component.scss:64]: checked, its fine

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 9 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/shared/header/header.component.scss:65]: checked, its fine

File: apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss (hunk 8 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/shared/header/components/new-group-account-sidebar/groups-account-sidebar.component.scss:6]: checked, its fine

File: apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.scss (hunk 6 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/reports/components/purchase-register-expand-component/purchase.register.expand.component.scss:50]: checked, its fine

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.scss [hunk 39]
```diff
@@ -1541,7 +1536,7 @@ label.sales-country-label {
     transform: translateY(-40px);
     width: 20px;
     height: 20px;
-    border: 1px solid $nobel;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.scss:1544]: checked, its fine

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.scss [hunk 40]
```diff
@@ -741,7 +736,7 @@ input[type="text"] {
 }
 
 a.copy-invoice {
-    color: $primary;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.scss:744]: checked, its fine

File: apps/web-giddh/src/app/onboarding/onboarding.component.scss [hunk 41]
```diff
@@ -208,7 +208,7 @@ a.text-dark {
 
 #talk_sales {
     padding-bottom: 2px;
-    border-bottom: 1px solid $silver;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/onboarding/onboarding.component.scss:211]: checked, its fine

File: apps/web-giddh/src/app/ledger/ledger.component.scss [hunk 42]
```diff
@@ -259,7 +260,7 @@ table.ledger-footer-1 td[class*="col-"] {
     text-align: center;
 }
 .pagination > li > a:hover {
-    color: $cinnabar-orange;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/ledger/ledger.component.scss:262]: checked, its fine

File: apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.scss [hunk 43]
```diff
@@ -73,18 +75,14 @@ ng-select .select-drop i.fa.fa-caret-up::before {
     background: transparent;
     border: none !important;
     padding: 5px;
-    color: $gray;
+    color: $silver;
 }
 
 .notfound-option.active {
-    background: $lavender !important;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.scss:80]: checked, its fine

File: apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.scss [hunk 44]
```diff
@@ -73,18 +75,14 @@ ng-select .select-drop i.fa.fa-caret-up::before {
     background: transparent;
     border: none !important;
     padding: 5px;
-    color: $gray;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.scss:76]: checked, its fine

File: apps/web-giddh/src/app/ledger/components/update-ledger-discount/update-ledger-discount.component.scss [hunk 45]
```diff
@@ -19,7 +19,7 @@
 .or-line {
     display: flex;
     align-items: center;
-    color: $gainsboro;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/ledger/components/update-ledger-discount/update-ledger-discount.component.scss:22]: checked, its fine

File: apps/web-giddh/src/app/ledger/components/advance-search/advance-search.component.scss [hunk 46]
```diff
@@ -54,7 +54,7 @@
         margin-bottom: 20px;
     }
     .amount-total {
-        background: $whisper-light;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/ledger/components/advance-search/advance-search.component.scss:57]: checked, its fine

File: apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.scss [hunk 47]
```diff
@@ -147,15 +147,15 @@
 
 .table-modal tbody td {
     border-top: 0px solid !important;
-    border-right: 1px solid $silver;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/invoice/eWayBill/create/eWayBill.create.component.scss:150]: checked, its fine

File: apps/web-giddh/src/app/inventory/jobwork/jobwork.component.scss (hunk 34 above)
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/inventory/jobwork/jobwork.component.scss:65]: checked, its fine

File: apps/web-giddh/src/app/accounting/accounting.component.scss [hunk 48]
```diff
@@ -137,7 +137,7 @@
     padding: 0 !important;
     border: 0;
     z-index: 0;
-    border-left: 1px solid $light-gray;
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/accounting/accounting.component.scss:140]: checked, its fine

File: apps/web-giddh/src/app/contact/contact.component.html [hunk 49]
```diff
@@ -527,13 +523,14 @@ <h1 class="mr-b1">{{localeData?.type_message_body}}:</h1>
...
                             <td class="position-relative word-wrap customer-name cursor-pointer"
-                                (clickOutside)="dropdown.hide()" (dblclick)="dropdown.show()" data-title="Customer">
+                                (clickOutside)="dropdown.hide()" (dblclick)="dropdown.show()"
+                                data-title="{{localeData?.customer}}">
                                 <div class="customer-name-filed">
                                     <span class="inline table-checkbox">
                                         <label></label>
                                         <input type="checkbox" (click)="selectAccount($event, account)"
                                             [(ngModel)]="account.isSelected"
-                                            [ngClass]="{'checkbox-visible': (selectedCheckedContacts.length || isMobileScreen), 'position-absolute': !selectedCheckedContacts.length}" />
```
[the remark this answers is not in the document]
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/contact/contact.component.html:536]: checked, its fine

**Review by shalinee-walkover — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**